snipeit.sh

```diff
@@ -288,9 +299,10 @@ case $distro in
 		curl -sS https://getcomposer.org/installer  | php
 		php composer.phar install --no-dev --prefer-source
 		perms
+		chown -R www-data:www-data "/var/www/$name"
```

Comment:
Ah, I guess you *do* have the `chown -R` in other branches too! Good, that's at least consistent. Are we still sure it's a good idea?